types/styled-components/test/index.tsx

```diff
@@ -16,6 +16,7 @@ import styled, {
     StyledComponent,
     ThemedStyledComponentsModule
 } from "styled-components";
+import {} from "styled-components/cssprop";
```

Comment:
1) a shorter syntax can be used `import "styled-components/cssprop";`
2) I think TS will insert a real import, that will break in run-time

to fix (2) I suggest recommend users to include it with `///<reference>` directive or add the file to their `tsconfig.json`.

Reply:
ah, ok, that makes sense, thank you!
what do you think about a recommendation to use `///` or add to tsconfig?

Reply:
How would it be added to tsconfig; `styled-components/cssprop` in the `types` array?